api/src/main/java/org/pmiops/workbench/db/model/Cohort.java

```diff
@@ -1,17 +1,19 @@
 package org.pmiops.workbench.db.model;
 
+import java.sql.Timestamp;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.Index;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
-import org.joda.time.DateTime;
```

Comment:
Does JODA time not work? It's the recommendation within Google (which I assume is why you used it initially). Or, I see you're using JODA time above; we just translate to SQL objects for storage?